api_v3/lib/types/enums/KalturaExportToCsvOptionsType.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+/**
+ * @package api
+ * @subpackage enum
+ */
+class KalturaExportToCsvOptionsType extends KalturaDynamicEnum implements ExportToCsvOptionsType
```

Comment:
The way you built it , it means it cannot be extended, meaning only one option can be set, and this does not make sense.
This shoud be chanegd to array of options.